cypress/support/elements/sage-modeler-elements.js

```diff
@@ -0,0 +1,112 @@
+const SageModelerElements = {
+    BTN_CREATE_NEW_DOCUMENT: '.modal-dialog-blocking-message .open-or-create div:nth-child(2) button',
+    BTN_OPEN_DOC_OR_BROWSE_EXAMPLES: '.modal-dialog-blocking-message .open-or-create div:nth-child(1) button',
```

Comment:
It may be better looking for the button that contains the test "Open Document" or "Create New Document" (ie. `(.modal-dialog-blocking-message .open-or-create button).contains(Open Document) and (.modal-dialog-blocking-message .open-or-create button).contains(Create New Document)`

Reply:
Deleted these two elements and using the previous "CfmObject" which i refactored to the current page elements style.